app/assets/stylesheets/content/_forms.sass

```diff
@@ -67,36 +67,35 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     .form--section
       padding-top: 0px
```

Comment:
I just noticed that there is a marign at the bottom of the `form--section`. For the danger-zone you might want to fixate a `margin: 0`? IMO this looks more appealing...